### ISSUE TITLE
Clicking on the Comment icon should open the idea modal

### ISSUE THREAD

**Issue by ikosovec** [ezl/dailyidea.com#845]
Steps:
- I log in
- Go to my profile (https://beta.dailyidea.com/profile/ivan)
- Two ideas on my profile
- Click on Comment button on the top one (public idea)
- Get redirected to https://beta.dailyidea.com/i/ax83di/test-idea-again?comment=1 and then right after (0.5 sec) to https://beta.dailyidea.com/i/ax83di/test-idea-again?category=new
- Go back to my profile
- Click on Comment on bottom idea (private idea)
- Get redirected to a 404 (https://beta.dailyidea.com/i/40k8ao/test-idea-with-no-body-text-to-check-if-there-s-still-nundefined?comment=1). I have covered this question in a separate design ticket #844

**Comment by ikosovec** [comment]
When I enter https://beta.dailyidea.com/i/ax83di/test-idea-again?comment=1 in my browser, it also redirects me to https://beta.dailyidea.com/i/ax83di/test-idea-again?category=new

**Comment by ikosovec** [comment]
Oh okay, I think I understand what's happening here, and it's probably **NOT** a bug!

Basically, when I opened DI, I didn't scroll through the onboarding cards on the home page. I went directly to log in and my profile. So these 2 URLs from my last comment were redirecting me to the onboarding cards. This looked like I was being redirected to the home page instead of the specific idea in question here - which looked like a bug to me.

Then I went through the onboarding cards and now I'm being redirected to the specific idea in question - which looks like the correct flow and what I was expecting.

**I still think we need some more work on this part.**
I have been using and testing DI for a couple of weeks and spent a lot of hours on the website - and I didn't understand what's going on here - I even thought it was a bug. This behaviour will be surely confusing for regular users if it was confusing for me (I consider myself a power user). We may need to re-think what we want here. What do you guys think? @ezl @vedmant @Watson-B

**Comment by ezl** [comment]
Agree -- this isn't idea. @Watson-B let's schedule time to talk about this and the tree.

**Comment by Watson-B** [comment]
@ikosovec yeah, it's confusing. I recreated the setup with my profile and I'm encountering the same thing. Adding to it, if I'm on https://beta.dailyidea.com/i/k3etgi/test?category=new and click the "New" tab, it does not take me to the top of the "new" stack, and instead keeps me on the same card. Showing a 404 when the user clicks on the comment icon on a private idea feels bad. One idea that I think can work is to repeat the pattern for disabling the share icon with the comment icon. Thoughts?
@ezl sounds good, I'll add it to my to-do list to bring up next time we talk. My schedule is flexible this week and next, so just let me know what works for you.

**Comment by ezl** [comment]
got it. I understand what you were seeing now, @ikosovec.

I didn't see that behavior. I thought you were referring to the problem of:

action: click to comment
outcome: go to idea page

as being the issue.

That is non-ideal. I have a separate question to address with @Watson-B that I have added to our conversation notes about re-evaluating how we want ALL the buttons on this summary page to work.

Since #844 already documents the other issue, I propose that we close this issue now regarding the bug on the comment page.

I've created a new card to stand as a placeholder for @Watson-B and I to also evaluate and be more explicit about WHEN we want the explainer cards to show up in general, which will also solve the edge case that you ran into: #850
